nixos/configuration.nix

```
@@ -109,7 +109,7 @@
 
   services.cage = {
     enable = true;
-    program = "${pkgs.google-chrome}/bin/google-chrome-stable --simulate-outdated-no-au=\"01 Jan 2099\" --kiosk \"http://localhost:8080\"";
+    program = "${pkgs.google-chrome}/bin/google-chrome-stable --disable-http-cache--simulate-outdated-no-au=\"01 Jan 2099\" --kiosk \"http://localhost:8080\"";
```

```suggestion
    program = "${pkgs.google-chrome}/bin/google-chrome-stable --disable-http-cache --simulate-outdated-no-au=\"01 Jan 2099\" --kiosk \"http://localhost:8080\"";
```
I think you need a space here...